pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
I just took a quick look into the issue and apparently this is caused due to GSdx being initialized even after a shutdown of the VM.